bin/main.rb

```diff
@@ -1,2 +1,37 @@
 #!/usr/bin/env ruby
-puts "Hello world!"
\ No newline at end of file
+puts "Hello world!"
+def show_board(
+    board = ["1 ", "2 ", " 3", "4 ", "5 ", "6 ", " 7", " 8", " 9"])
+      puts 'show board'
+    #board output
+    puts " #{board[0]} | #{board[1]} | #{board[2]} "
+    puts "-----------"
+    puts " #{board[3]} | #{board[4]} | #{board[5]} "
+    puts "-----------"
+    puts " #{board[6]} | #{board[7]} | #{board[8]} "
+end
+
+board = ["1 ", "2 ", " 3", "4 ", "5 ", "6 ", " 7", " 8", " 9"]
+#puts show_board([1,2,3,4,5,6,7,8,9])
+puts "Welcome to the game! who will you be today? X or O?"
+player1 = gets.chomp
+player1.upcase!
+
+if player1 == "X"
+    player2 = "O"
+    puts "player 1 is #{player1} and player 2 is #{player2}."
+
+else
+    player2 = "X"
+    puts "player 1 is #{player1} and player 2 is #{player2}."
+end
+
+puts "In what numbered cell would you start from?"
+input = gets.chomp
+input.upcase!
+turn = Integer(input)
+board[turn-1] = player1
+puts `clear`
+
+show_board(board)
+    
```

Comment:
there is nothing required for us to do after we display the updated board, according to what milestone 2 has outlined.

Reply:
Dear Mael Fosso,

From the changes required comment you left, all the things you listed are meant to be implemented in the 3rd mile stone, which is game logic. This is clearly explained in the pathright, and as such I feel they are out of the scope of the 2nd milestone. Kindly review this.

Reply:
@Audrey-Ella-xo you are right they should be implemented in the 3rd milestone but in that milestone you have to write the mock up of those part.

I mean, 
- where are you going to check for input validation ? 
- how will you handle the loop ?
- where are you going to check for the state of the game (draw or win) ?

From that [Milestone 2 description](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/) It said 
- Display prompts for users (hint: puts/print), collect user input (hint: get.chomp) and only mock the basic flow of the game (you can use puts or inline comments to say for example “Now, your move is displayed on the board”).
- The final point of this milestone is to make sure that you thought about **all actions needed** to play Tic Tac Toe. **It does not need to work yet** - this will be done in the next milestone